Harvest.Web/ClientApp/src/Projects/AdhocProject.tsx

```diff
@@ -142,10 +146,13 @@ export const AdhocProject = () => {
         )
     );
 
+    quote.activities = activities.filter((a) => a.workItems.length > 0);
```

Comment:
You should never set a property on a state variable itself, that's what the setter is for

Reply:
@srkirkland so :
 const quote: QuoteContent = {
      projectName: project.name,
      acres: 0,
      acreageRate: 0,
      years: 0,
      acreageTotal: 0,
      activitiesTotal: 0,
      laborTotal: 0,
      equipmentTotal: 0,
      otherTotal: 0,
      grandTotal: 0,
      fields: [],
      activities: activities.filter((a) => a.workItems.length > 0),
      acreageRateId: null,
      acreageRateDescription: ""
    }

Reply:
sure or just use `const quote = new QuoteContentImp();` and then set activities from there, assuming it sets the defaults you want